modules/extensions/k8stools.tf

```diff
@@ -23,14 +23,23 @@ resource "null_resource" "install_k8stools_on_operator" {
     content     = local.install_helm_template
     destination = "/home/opc/install_helm.sh"
   }
+
+  provisioner "file" {
+    content     = local.install_kubectx_template
+    destination = "/home/opc/install_kubectx.sh"
+  }
+
   provisioner "remote-exec" {
     inline = [
       "chmod +x $HOME/install_kubectl.sh",
       "$HOME/install_kubectl.sh",
       "rm -f $HOME/install_kubectl.sh",
```

Comment:
You probably should check that `$HOME` contains a value that is meaningful otherwise the `rm -f` may run against `/install_kubectl.sh`.

Reply:
for example?

Reply:
Something like this would test that the file exists before attempting to either run it or delete it.

```bash
if [ -f "$HOME/install_kubectl.sh" ]; then bash $HOME/install_kubectl.sh; fi
```

In the case of your `rm` command, something like:

```bash
if [ -f "$HOME/install_kubectl.sh" ]; then rm -f $HOME/install_kubectl.sh; fi
```